### DISCUSSION TITLE
Feedback - Disabled Actions after "Design improvements to GitHub Actions navigation"

### DISCUSSION THREAD

**Discussion by michaelmworthington** [community/community#36168] [Actions]
After the UI Refresh, the main page of the repository's actions no longer shows an indicator icon when a workflow is disabled.

We use GitHub actions to deploy to environments and periodically need to stop deployments. The new UI now requires a user to click on each workflow to determine which ones are disabled and which ones are running.

**Comment by jdgirald** [comment]
same issue, makes really difficult to maintain workflows. Please add the indicator icon when a workflow is disabled back
